common/lib/xmodule/xmodule/js/src/video/09_poster.js

```diff
@@ -25,7 +25,10 @@ define('video/09_poster.js', [], function () {
         template: _.template([
             '<div class="video-pre-roll is-<%= type %> poster" ',
                 'style="background-image: url(<%= url %>)">',
-                '<button class="btn-play">', gettext('Play video'), '</button>',
+                '<button class="btn-play btn-pre-roll">',
+                    '<img src="/static/images/play.png" alt="">',
```

Comment:
Empty `alt` attribute because we have inline text that immediately follows this image.